src/Data/Nat/Base.agda

```diff
@@ -130,6 +132,47 @@ instance
 >-nonZero⁻¹ : ∀ n → .{{NonZero n}} → n > 0
 >-nonZero⁻¹ (suc n) = z<s
 
+-- The property of being a non-zero, non-unit
+
+pattern 2+ n = suc (suc n)
+
+trivial : ℕ → Bool
+trivial 0      = true
+trivial 1      = true
+trivial (2+ _) = false
+
+NonTrivial : Pred ℕ 0ℓ
+NonTrivial = T ∘ not ∘ trivial
+
+instance
+  nonTrivial : ∀ {n} → NonTrivial (2+ n)
+  nonTrivial = _
+
+pattern 1<2+n {n} = s<s (z<s {n})
+
+-- Constructors
+
+n>1⇒nonTrivial : ∀ {n} → 1 < n → NonTrivial n
+n>1⇒nonTrivial 1<2+n = _
+
+nonZero⇒≢1⇒nonTrivial : ∀ {n} → .⦃ NonZero n ⦄ → n ≢ 1 → NonTrivial n
```

Comment:
Everywhere else in the library I have used the non-unicode version of the instance brackets. Two advantages, shorter to type, and more obvious how to type. Could we be consistent and stick with the double braces everywhere? (Should be added to the style guide)

Reply:
See also #2197 , where I have folded this in as a commit to `style-guide`...